docs/issue_tracking.md

```diff
@@ -2,62 +2,137 @@
+* Active - it is being worked on.
+* Closed - issue is closed
 
-The current target date for hitting a milestone is in the "Target" field.
+### Epic
 
-The current list of milestones can be seen on the [Milestones
-view](https://github.com/orgs/llvm/projects/4/views/15). The [Roadmap
-view](https://github.com/orgs/llvm/projects/4/views/12) also provides a
-graphical view of the milestones over time.
+An "Epic" is our name for some large piece of functionality we're working
+towards.  All work items associated with an Epic have this field set.
 
-The main body for a Milestone issue follows this format:
+### Kind
 
-* Preamble - text providing information about the milestone, what the goals are,
-  what the challenges might be, why it is an interesting milestone etc.
-* Workstreams - a list of workstreams, and the workstream milestones that
-  contribute to this one.
-* Postnote - additional text after the workstream list.
+Issues can be:
+* Scenario
+* Deliverable
+* Task
+* Bug
 
-The Workstreams section starts with the heading `## Workstreams` and ends either
-at the end of the body, or at a horizontal rule (`---`). The contents of this
-section is entirely derived from the Workstream issues.
+Note that we don't use githubs new "Type" field because that isn't necessarily
+consistent across repos.
 
-## Workstreams
+More details on what the different kinds of issues are can be found below.
 
-The work involved in adding HLSL Support has been split into a number of
-workstreams. The [Workstreams
-view](https://github.com/orgs/llvm/projects/4/views/16) shows a list of current
-workstreams.
+### Iteration
 
-A workstream issue can be identified by the "Category" field set to "Workstream".
+The iteration field specifies when we expect an issue to be closed. 
 
-The workstream issue is formatted similarly to a milestone issue:
+### Estimate
 
-* Preamble
-* Milestones
-* Postnote
+This is a rough estimate of how long an issue will take to complete.  Values
+include:
 
-The Milestones section starts with the header `## Milestones`. This section is the authority for what gets aggregated into the milestone issues.
+* Hours, Days, Week, Sprint, Multiple Sprints
+* Break Up - this estimate is set to indicate that we think this issues need to
+  be broken up into smaller chunks
+* Umbrella - this indicates that this issue is actually an "umbrella" one that
+  exists to hold multiple sub-issues that are estimated separately.
+* n/a - it doesn't always make sense to provide an estimate for an issue, in
+  which case we can set it to n/a.
+
+### Blocked
+
+Issues can be blocked for some different reasons:
+
+* PR - waiting for a PR to complete
+* Refinement - waiting for agreement that it is ready to go from Planning to
+  Ready
+* Blocked - waiting for some other dependency before work on this one can
+  continus.
+
+When marking an issue as blocked we try and add a comment on the issue
+explaining why.
```

Comment:
The "PR" and "Refinement" blocked states explain themselves, so I assume that this is saying we do this when we use the generic "Blocked", but that isn't entirely clear to me from the text.

Reply:
Updated to:

> When marking an issue as "Blocked" we try and add a comment on the issue
explaining why.